pkg/proxy/iptables/proxier.go

```diff
@@ -280,7 +280,8 @@ func NewProxier(ipt utiliptables.Interface,
 
 	// Generate the masquerade mark to use for SNAT rules.
 	masqueradeValue := 1 << uint(masqueradeBit)
-	masqueradeMark := fmt.Sprintf("%#08x/%#08x", masqueradeValue, masqueradeValue)
+	masqueradeMark := fmt.Sprintf("%#08x", masqueradeValue)
+	klog.V(2).Infof("iptables(%s) masquerade mark: %s", ipt.Protocol(), masqueradeMark)
```

Comment:
(FTR the second line wasn't part of 92035, it was something that existed in master but not release-1.18, so maybe shouldn't have been included in this backport)

Reply:
Removed the extra line